crates/nomination/src/lib.rs

```diff
@@ -292,8 +292,8 @@ impl<T: Config> Pallet<T> {
     }
 
     pub fn get_total_nominated_collateral(vault_id: &T::AccountId) -> Result<Collateral<T>, DispatchError> {
-        let vault_backing_collateral = ext::vault_registry::get_backing_collateral::<T>(vault_id)?;
-        let vault_actual_collateral = ext::vault_registry::compute_collateral::<T>(vault_id)?;
+        let vault_backing_collateral = ext::vault_registry::get_vault_total_collateral::<T>(vault_id)?;
+        let vault_actual_collateral = ext::vault_registry::get_vault_collateral::<T>(vault_id)?;
```

Comment:
```suggestion
        let vault_only_collateral = ext::vault_registry::get_vault_collateral::<T>(vault_id)?;
```

Maybe that makes it a bit more explicit?